pypeit/coadd2d.py

```diff
@@ -895,26 +897,29 @@ def snr_report(self, snr_bar, slitid=None):
         msg_string += msgs.newline() + '-------------------------------------'
         msgs.info(msg_string)
 
-    def offsets_report(self, offsets, offsets_method):
+    def offsets_report(self, offsets, platescale, offsets_method):
         """
         Print out a report on the offsets
 
         Args:
-            offsets:
-            offsets_method:
-
+            offsets (`numpy.ndarray`_)
+                Array of offsets
+            platescale (float):
+                the plate scale for this detector
+            offsets_method (str):
+                A string describing the method used to determine the offsets
         Returns:
 
         """
 
         if offsets_method is not None and offsets is not None:
-            msg_string = msgs.newline() + '---------------------------------------------'
+            msg_string = msgs.newline() + '---------------------------------------------------------------------------------'
```

Comment:
You know that you can also the `format` method provided by python such that you don't have to hard-code the lines? :)